packages/storybook-utils/src/index.css

```diff
@@ -0,0 +1,25 @@
+@import url("../../sit-onyx/src/styles/variables-onyx.css");
+
+.onyx-disclaimer {
+  border-radius: 8px;
+  padding: 16px;
+  line-height: 24px;
+  color: rgba(60, 60, 67);
+  background-color: rgba(100, 108, 255, 0.14);
```

Comment:
is it intentionally not linked to the css variables?
(looking in the future, a theme switch might want to change the value?)

Reply:
Nope, I changed it